rebasehelper/checker.py

```diff
@@ -30,7 +30,15 @@
 
 
 class BaseChecker(object):
-    """ Base class used for testing tool run on final pkgs. """
+    """Base class used for testing tool run on final pkgs.
+
+    Attributes:
+        NAME(str): Name of the checker
+        DEFAULT(bool): Boolean value making the checker run by default, if no checker settings were set.
+        category(str): Checker category which enables running only checkers of a certain category in various stages
+        of the rebase.
+        results_dir(str): path to the rebase-helper-results/checkers/ directory
+    """
```

Comment:
Sorry for not commenting on this for so long. 

I don't think this docstring is very useful. I would say that `Base class of package checkers.` sounds way better than `Base class used for testing tool run on final pkgs.`. The attributes are mostly self-explanatory, and `results_dir` definitely shouldn't be described as an actual path (which can potentially change).

Reply:
Do it if you want, but make the description short and meaningful.

Description of `results_dir` shouldn't be a path, no matter how specific, it should say what is the attribute for. What is it for anyway? I don't seem to see a reason for it.

And the other attributes are inconsistent in capitalization, why is `category` lower-case?

Reply:
Alright. Category is lower case by a mistake, so I will add a commit for that as well.

Reply:
I'm not sure if this is supposed to be finished or not, as it is really inconsistent now. Anyway, I would still adjust some of the descriptions:

```
"""Base class of package checkers.

Attributes:
    NAME(str): Name of the checker.
    DEFAULT(bool): If True, the checker is run by default.
    category(str): Category which determines the phase in which the checker is run.
    results_dir(str): Path where the results are stored.
"""
```

Also there could be a list of valid categories.